tests/testStatsdClient.cpp

```diff
@@ -1,37 +1,65 @@
 #include <StatsdClient.hpp>
 #include <cassert>
+#include <iostream>
 
 using namespace Statsd;
 
-int main() {
-    StatsdClient client{"localhost", 8080, "myPrefix.", 20};
+int main(int argc, char** argv) {
+    // connect to a rubbish port and make sure initialization failed
+    StatsdClient client{"256.256.256.256", 8125, "myPrefix.", 20};
+    if (client.errorMessage().empty()) {
+        std::cerr << "should not be able to connect to ridiculous ip" << std::endl;
+        return EXIT_FAILURE;
+    }
 
     // set a new config with a different port
-    client.setConfig("localhost", 8000, "myPrefix.");
+    client.setConfig("localhost", 8125, "myPrefix.");
+    if (!client.errorMessage().empty()) {
+        std::cerr << client.errorMessage() << std::endl;
+        return EXIT_FAILURE;
+    }
 
     // Increment "coco"
     client.increment("coco");
-    assert(!client.errorMessage());
```

Comment:
i figured that also release builds should fail the tests so rather than going with `asserts` i just check the values and return failure.